src/main/controllers/ContactTheTribunalController.ts

```diff
@@ -17,25 +18,15 @@ export default class ContactTheTribunalController {
       ...req.t(TranslationKeys.CONTACT_THE_TRIBUNAL, { returnObjects: true }),
     };
 
-    const applicationsAccordionItems = applications.map(application => {
-      const label = translations.sections[application].label;
-      return {
-        heading: {
-          text: label,
-        },
-        content: {
-          html:
-            "<p class='govuk-body'>" +
-            translations.sections[application].body +
-            '</p> <br>' +
-            "<a class='govuk-link govuk-body' href='/contact-the-tribunal/" +
-            application +
-            "'>" +
-            label +
-            '</a>',
-        },
-      };
-    });
+    const userCase = req.session.userCase;
+    const applicationsAccordionItems = generateAccordionItems(applications, translations);
+    const notifications = userCase?.sendNotificationCollection?.filter(
+      it => it.value.sendNotificationCaseManagement === 'Request'
+    );
+
+    if (!notifications?.length) {
+      applicationsAccordionItems.splice(applications.length - 1, 1);
```

Comment:
This looks like a refactor, splitting out the generate logic into its own method, much cleaner, but I'm not sure I follow the addition of removing the last item in `applicationsAccordionItems` if there are no request notifications. Is this a bug fix?

Reply:
The last item in the list is the submit documents for a hearing section which is part of this ticket, but Marie wants this section to be viewable only after there has been a notification from the tribunal to request documents.